lib/music_today_api_wrapper.rb

```diff
@@ -9,4 +9,9 @@ def self.configure
     self.configuration ||= Configuration.new
     yield(configuration)
   end
+
+  def self.products
+    product_services = ProductServices.new
+    product_services.all_products
```

Comment:
allign this call with the method name. Would be nice if you tested the service method :)